main/install/configuration.dist.php

```diff
@@ -2347,6 +2347,22 @@
 //CREATE INDEX c_attendance_sheet_user ON track_e_access_complete (attendance_sheet_id, user_id);
 //$_configuration['attendance_allow_comments'] = false;
 
+// Enable categories in tool Wiki.
+// Requires DB changes:
+/*
+CREATE TABLE c_wiki_rel_category (wiki_id INT NOT NULL, category_id INT NOT NULL, INDEX IDX_AC88945BAA948DBE (wiki_id), INDEX IDX_AC88945B12469DE2 (category_id), PRIMARY KEY(wiki_id, category_id)) DEFAULT CHARACTER SET utf8 COLLATE `utf8_unicode_ci` ENGINE = InnoDB;
```

Comment:
This code was generated by the doctrine command.
The code related to collate and engine exists in other code that creates tables before this PR.
The indexes names are the same that the names created when chamilo is installed and the database schema es created.

Reply:
This will all end in tears...
Just kidding, I know we've let Doctrine deal with a series of queries. I'm not sure why Doctrine does it like that but hopefully we improve this in the future.